usaspending_api/download/v2/download_count.py

```diff
@@ -52,15 +64,33 @@ def post(self, request):
             options["time_period_obj"] = new_awards_only_decorator
             filter_query = QueryWithFilters.generate_transactions_elasticsearch_query(filters, **options)
             search = TransactionSearch().filter(filter_query)
-            total_count = search.handle_count()
+            total_transaction_count = search.handle_count()
+        else:
+            options = {}
+            time_period_obj = AwardSearchTimePeriod(
+                default_end_date=settings.API_MAX_DATE, default_start_date=settings.API_SEARCH_MIN_DATE
+            )
+            new_awards_only_decorator = NewAwardsOnlyTimePeriod(
+                time_period_obj=time_period_obj, query_type=_QueryType.TRANSACTIONS
+            )
+            options["time_period_obj"] = new_awards_only_decorator
+            filter_query = QueryWithFilters.generate_transactions_elasticsearch_query(filters, **options)
+            search = AwardSearch().filter(filter_query)
+            total_award_count = search.handle_count()
 
-        if total_count is None:
-            total_count = 0
+        if total_transaction_count is None:
+            total_transaction_count = 0
+        elif total_subaward_count is None:
+            total_subaward_count = 0
+        elif total_award_count is None:
+            total_award_count = 0
 
         result = {
-            "calculated_transaction_count": total_count,
+            "calculated_transaction_count": total_transaction_count,
+            "calculated_award_count": total_award_count,
+            "calculated_subaward_count": total_subaward_count,
             "maximum_transaction_limit": settings.MAX_DOWNLOAD_LIMIT,
-            "transaction_rows_gt_limit": total_count > settings.MAX_DOWNLOAD_LIMIT,
+            "transaction_rows_gt_limit": total_transaction_count > settings.MAX_DOWNLOAD_LIMIT,
             "messages": get_generic_filters_message(
```

Comment:
I missed adding the A/C of adding the deprecation message to this endpoint, because `subawards` isn't a valid filter according to the API contract for this endpoint.

@ayubshahab this is the deprecation message for `subawards` that I added to the other tickets:
> 'subawards' will be deprecated in the future. Set ‘spending_level’ to ‘subawards’ instead. See documentation for more information.

Reply:
Message has been added.

Reply:
@ayubshahab I don't see any new messages in the response from the API